.github/workflows/gallery.yml

```diff
@@ -8,21 +8,33 @@ on:
       - master
 permissions:
   contents: write
-
+  id-token: write
+  pages: write
 jobs:
   test:
     name: Publish Gallery to Github Pages
     if: github.repository != 'gautamkrishnar/github-pages-gallery'
     runs-on: ubuntu-latest
     steps:
+      - name: Setup GitHub Pages ⚙️
+        uses: actions/configure-pages@v3.0.4
       - name: Checkout 🛎️
         uses: actions/checkout@v3
         with:
           persist-credentials: false
       - name: Build Gallery 🔧
         run: docker run -v "$(pwd):/work" thumbsupgallery/thumbsup /bin/sh -c "cd /work/ && thumbsup --config config.json"
+      - name: Package and upload artifact 📦
+        uses: actions/upload-pages-artifact@v1.0.7
+        with:
+          path: 'build_output'
+          name: gh-pages
```

Comment:
Fix the error `Error: No uploaded artifact was found! Please check if there are any errors at build step, or uploaded artifact name is correct.`